src/main/java/io/vertx/mqtt/MqttEndpoint.java

```diff
@@ -41,6 +44,31 @@
   void close();
 
   /**
+   * @return the remote address for this socket
+   */
+  @CacheReturn
+  SocketAddress remoteAddress();
```

Comment:
Exposing `remoteAddress` is not related to the purpose of this PR on SSL right ? Why do you need this ?

Reply:
It is useful for logging purposes. I can create another PR if you wish for that functionality.